i18n/de.json

```
@@ -48,13 +48,13 @@
     "Expires":
         "Ablaufzeit",
     "Burn after reading":
-        "Einmal-Text",
+        "Nach dem lesen löschen",
```

```suggestion
        "Nach dem Lesen löschen",
```

Substantiviertes Verb, sollte man schon groß schreiben. 
Was ich auch noch schwierig finden könnte, wäre, dass das evt. etwas lang wird in der Benutzeroberfläche.